site.yml

```diff
@@ -1,8 +1,14 @@
 ---
+- name: gather facts on all hosts
+  hosts: all
+  tags: always
+  become: no
+  tasks:
+    - setup:
```

Comment:
So this still needs the trick where you're delegating the fact gathering, and you're doing it over every host in the inventory. The intent is that we get facts gathered for every host in the inventory (even if called with --limit).